python/shotgun_data/shotgun_data_retriever.py

```diff
@@ -1068,25 +1068,32 @@ def _on_task_completed(self, task_id, group, result):
             return
 
         action = result.get("action")
-        if action in ["find", "find_one", "create", "delete", "update"]:
-            self.work_completed.emit(str(task_id), action, {"sg":result["sg_result"]})
-        elif action == "nav_expand":
-            self.work_completed.emit(str(task_id), "nav_expand", {"nav":result["nav_result"]})
+        if action in ["find", "find_one", "create", "delete", "update", "nav_expand"]:
```

Comment:
`work_completed` used to emit `{"nav":result["nav_result"]}`but now emits `{"sg": result["sg_result"]}`, won't this break backwards compatibility?